java/androidpayload/app/pom.xml

```diff
@@ -1,11 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<groupId>com.metasploit</groupId>
 	<artifactId>Metasploit-AndroidPayload</artifactId>
 	<version>1-SNAPSHOT</version>
 	<packaging>apk</packaging>
 	<name>AndroidPayload for Metasploit</name>
+	<parent>
```

Comment:
Specifically this is aimed to pull in the 1.5 compiler configuration to avoid the cafebabe compilation error

https://github.com/rapid7/metasploit-payloads/blob/a76937ebd0b83e2ef7f5e0bee34ef8b9c6203e4f/java/pom.xml#L24-L34